backend/ee/danswer/external_permissions/confluence/doc_sync.py

```diff
@@ -362,22 +230,29 @@ def confluence_doc_sync(
     it in postgres so that when it gets created later, the permissions are
     already populated
     """
-    confluence_client = build_confluence_client(
-        connector_specific_config=cc_pair.connector.connector_specific_config,
-        credentials_json=cc_pair.credential.credential_json,
+    confluence_connector = ConfluenceConnector(
+        **cc_pair.connector.connector_specific_config
     )
+    confluence_connector.load_credentials(cc_pair.credential.credential_json)
```

Comment:
probably should check a return value rather than some internal state afterwards.